### PR TITLE
feat(theme-default): update FontAwesome to 5.14.0

### PR DESCRIPTION
**Pull request by dogatana**
This is a pull request to change theme-default.

I'm considering using honkit as an authoring tool for my project.
Since it uses FontAwesome 5 icons in the text, so I really need this feature.

Thank you in advance.

### CODE REVIEW

File: packages/@honkit/theme-default/src/less/website.less [hunk 1]
```diff
@@ -1,5 +1,8 @@
 @import "base/all.less";
-@import "../../font-awesome/less/font-awesome.less";
+@import "../../../../../node_modules/@fortawesome/fontawesome-free/less/fontawesome.less";
```
Comment (azu) [inline packages/@honkit/theme-default/src/less/website.less:2]: Does `less` support module name resolution like the following?

```suggestion
@import "@fortawesome/fontawesome-free/less/fontawesome.less";
```

**Comment by azu** [comment]
Preview HonKit Document site: https://deploy-preview-132--honkit.netlify.app/

I'm not familiar with font-awesome.
Is there a BREAKING CHANGE between 4 and 5?

- [Upgrading from Version 4 | Font Awesome](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4)

**Comment by dogatana** [comment]
> Is there a BREAKING CHANGE between 4 and 5?

As you posted, technically there is.

theme-default uses FontAwesome **ONLY** for website.css, so this change may affect website.

As far as I know, page header uses one icon, bars, for the button to show/hide side menu.
And some plugins, like fontsettings, back-to-top-button or page-toc-button,  use icons as well.
We can use icons in README.md.
They all may be affected.

If it's hard to merge this, I'd like to use locally.
I highly appreciate if you could let me know how to do it.

I tried npm, but could not get through. 😢

**Comment by dogatana** [comment]
I overlooked your comment.

> (Personally, I want to remove font-awesome from default theme #57 )

It seems good. In that case we can choose if we use FontAwesome or not.
A problem is that honkit and some plugins depend on it.

I've already forked and modified some plugins to customize the icons that they use.

**Comment by dogatana** [comment]
According to some considerations, I drop this PR.

The reason is that this modification depends on the internal structure of honkit.
As a result, it works only for locally installed honkit.

I'll take another approach to use FontAwesome 5.

**Pull request closed without merging**